src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -457,6 +457,12 @@ typedef VOID (*RtcOnDataChannel)(UINT64, struct __RtcDataChannel*);
  */
 typedef VOID (*RtcOnIceCandidate)(UINT64, PCHAR);
 
+/*
+ * IceSetInterfaceFilterFunc is fired when a callback function to filter network interfaces is assigned.
+ * The callback function is expected to check for specific interface names to be whitelisted/blacklisted
+ */
+typedef BOOL (*IceSetInterfaceFilterFunc) (PCHAR);
```

Comment:
this callback should also have a UINT64 as its custom data like other callbacks

Reply:
Even if I have a custom data, I am not sure how I will be dealing with it in the APIs that are not exposed to the user. Can you tell me how custom data would be consumed in the API call?

Reply:
I can add that. However, what do I pass in custom data? Any useful piece of information you can think of?